shortlister/view.py

```diff
@@ -20,11 +20,20 @@ def view_applicant_details(self, applicant: Applicant):
         """Prints details of applicant to console."""
         print(f"Details for {applicant.name}: ")
         print(f"CV Path: {applicant.cv}")
-        print("Scores:")
-        if applicant.scores is not None:
+        print("Scores:", end=" ")
```

Comment:
Move this header to inside the condition if scores exist (i.e. simply print "No scores" if no scores present)